skfda/exploratory/visualization/_parametric_plot.py

```diff
@@ -28,64 +31,69 @@ class ParametricPlot:
             a dim_codomain = 1, the fdata2 will be needed.
         fdata2: optional functional data set, that will be needed if the fdata1
             has dim_codomain = 1.
+        chart: figure over with the graphs are plotted or axis over
+            where the graphs are plotted. If None and ax is also
+            None, the figure is initialized.
+        fig: figure over with the graphs are plotted in case ax is not
+            specified. If None and ax is also None, the figure is
+            initialized.
+        ax: axis where the graphs are plotted. If None, see param fig.
     """
 
     def __init__(
         self,
         fdata1: FData,
         fdata2: Optional[FData] = None,
+        chart: Union[Figure, Axes, None] = None,
+        *,
+        fig: Optional[Figure] = None,
+        axes: Optional[Axes] = None,
     ) -> None:
+        BasePlot.__init__(self)
         self.fdata1 = fdata1
         self.fdata2 = fdata2
 
+        if self.fdata2 is not None:
+            self.fd_final = self.fdata1.concatenate(
+                self.fdata2, as_coordinates=True,
+            )
+        else:
+            self.fd_final = self.fdata1
+
+        self._set_figure_and_axes(chart, fig, axes)
+
     def plot(
         self,
-        chart: Union[Figure, Axes, None] = None,
-        *,
-        fig: Optional[Figure] = None,
-        ax: Optional[Axes] = None,
-        **kwargs: Any,
+        **kwargs,
```

Comment:
I am not sure how I feel about passing `kwargs` directly to `plot`? Are we doing that for other plots? I think I prefer to be more explicit and restrictive here.

Reply:
if they can be stored in a variable i can ask for them in the init

Reply:
I meant to use explicit parameters like `group`, `group_colors`, etc, instead of allowing everything without checking if that makes sense.